### ISSUE TITLE
[Task] 508 Compliance Review

### ISSUE THREAD

**Issue by sumiat** [HHS/grants-equity#473]
### Summary

We have prepared for 508 compliance for the beta site. We have: 
- Code linters tell us when we're missing a11y stuff
- We've run Google Lighthouse — 100% score
- USWDS is a 508-compliant design system ([accessibility docs](https://designsystem.digital.gov/documentation/accessibility/)) which we used

There is a formal approval process for 508 compliance that includes the following: 
* Form is submitted to the compliance team (508Helpdesk@hhs.gov), cc Emily & Lucas
* Team takes ~2 weeks to respond (more lead time better) 

This ticket is track that work and ensure we go through the review process before launch. 

### Acceptance Criteria
- [x] Complete any checklist items provided by HHS
- [x] Submit the compliance web form (submitted on Sep 14)
- [x] HHS & OS Accessibility Service Desk has responded with their evaluation 
- [x] Create any necessary remediations tickets (if needed) 
- [x] 508 compliance approval process is defined in internal wiki
